### ISSUE TITLE
Loss not converging

### ISSUE THREAD

**Issue by jl1990** [werner-duvaud/muzero-general#95]
Hi,

I am trying to train tictactoe as a test. As it seems to be the simplest model.
Although, even with an RTX 2080 TI for two hours and after tuning hyper parameters I don't seem to get a good model and loss does not seem to be reducing too much.

Am I missing something? How long does it take to train the model for a simple game like tic tac toe?

Thanks

**Comment by werner-duvaud** [comment]
Hi,

For me it works well.
To test you can set the value of `opponent` to `random` to see in tensorboard the performance of muzero against a random player.
Another parameter is the `ratio`, the performance depends on the selfplayed game part (cpu) vs training (GPU). Below are my results with 1h30 of training (intel I7 & GTX1050Ti Max-Q). You can see that muzero reward is increasing and that the opponent loses most of his games. It's not perfect but I played against him and he blocks most of the wins. My ratio is 0.27 if you want to fix this hyperparameter like me.

![Screenshot_2020-12-24 TensorBoard(7)](https://user-images.githubusercontent.com/40442230/103112653-1dbda900-4657-11eb-9606-0316b05f77f5.png)
![Screenshot_2020-12-24 TensorBoard(6)](https://user-images.githubusercontent.com/40442230/103112657-1f876c80-4657-11eb-928a-7832425c60c6.png)
![Screenshot_2020-12-24 TensorBoard(5)](https://user-images.githubusercontent.com/40442230/103112658-21513000-4657-11eb-8336-64baff0b35e0.png)

**Comment by jl1990** [comment]
After enabling tensorboard, I was also able to see most of the losses decreasing.
But still after 20000 games at tic tac toe, the AI level is still very low. 
I guess it needs much more training, but I cannot even imagine the number of games that will be needed for a more complex game.

**Comment by antoniosap** [comment]
same here. have GeForce GTX 1070 (8116 MB) for 48 hours. connect4 game. the loss function stagnates

**Comment by werner-duvaud** [comment]
The default hyperparameters are not necessarily optimal at all.  Also fixing the ratio as said previously already makes it possible to shorten the time.
